warehouse/query-core/src/main/java/datawave/query/iterator/QueryOptions.java

```diff
@@ -1448,6 +1460,16 @@ public boolean validateOptions(Map<String,String> options) {
             }
         }
         
+        if (options.containsKey(MATCHING_FIELD_SETS)) {
+            String matchingFieldSets = options.get(MATCHING_FIELD_SETS);
+            for (String fieldSet : Splitter.on(',').omitEmptyStrings().trimResults().split(matchingFieldSets)) {
+                String[] fields = Iterables.toArray(Splitter.on('=').omitEmptyStrings().trimResults().split(fieldSet), String.class);
+                if (fields != null) {
```

Comment:
fields can't be null, but it could have length == 0.

Reply:
ah, right.... thanks